doc/api/v1.yaml

```diff
@@ -67,17 +67,14 @@ paths:
                           format: '[0-9a-f]+'
                         reason:
                           oneOf:
-                            - description: This fragment was already processed by this node
+                            - description: |
+                                * `FragmentAlreadyInLog` -- this fragment was already processed by this node.
+                                * `FragmentInvalid` -- this fragment failed validation.
+                                * `PreviousFragmentInvalid` -- one of the previous fragments was rejected and `fail_fast` is enabled.
```

Comment:
Much more concise, just one case of split personality now.
This is perhaps the best we can get from how serde and serde-yaml generate serialization of enums.

The JSON-friendly way would be:

```json
[
    {
        "id": "fafadeadbeef012345678",
        "reason": "FragmentAlreadyInLog"
    },
    {
        "id": "badc0ffee987654321",
        "reason": "PoolOverflow",
        "pool_number": 2
    }
]
```

BTW, don't we have blockchain-wise pool IDs?

Reply:
What do you mean exactly? This refers to mempool (which is basically the same as the leader ID). This is something specific to a particular node deployment.